From: Sophie Brun <sophie@offensive-security.com>
Date: Fri, 18 Jun 2021 13:42:40 +0200
Subject: Remove Werror and add default CFLAGS and LDFLAGS

---
 configure | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/configure b/configure
index 2bc4643..3934465 100755
--- a/configure
+++ b/configure
@@ -68,7 +68,7 @@ SNOOP_SUPPORT=""
 CC_CHECK=""
 LIBDIRS=`$CAT /etc/ld.so.conf /etc/ld.so.conf.d/*.conf 2> /dev/null`
 OS_SYSTEM=`$UNAME -s 2> /dev/null`
-XCFLAGS="-g -O2 -Wall -Werror -I."
+XCFLAGS="-g -O2 -Wall -I."
 XCFLAGS_EXTRA=""
 XLDFLAGS=""
 XSHARED_LIBS=""
@@ -391,8 +391,8 @@ VERSION=$SFUZZ_VERSION
 MD5SUM=$MD5SUM
 INSTALL=$HAS_INSTALL
 INSTALL_DIR=$HAS_INSTALL_DIR
-CFLAGS=$XCFLAGS $XCFLAGS_EXTRA
-LDFLAGS=$XLDFLAGS
+CFLAGS+=$XCFLAGS $XCFLAGS_EXTRA
+LDFLAGS+=$XLDFLAGS
 TARGET_PLAT=$OS_SYSTEM
 CP=$CP
 SED=$SED
